lib/hpcbase.pm

```diff
@@ -5,24 +5,23 @@ use testapi;
 use utils 'systemctl';
 
 sub enable_and_start {
-    my ($self, $arg) = @_;
+    my $arg = shift;
     systemctl "enable $arg";
     systemctl "start $arg";
 }
 
 sub upload_service_log {
-    my ($self, $service_name) = @_;
-    testapi::script_run("journalctl -u $service_name > /tmp/$service_name");
-    testapi::script_run("cat /tmp/$service_name");
-    testapi::upload_logs("/tmp/$service_name");
+    my $service_name = shift;
```

Comment:
I consider `my ($service_name) = @_;` a safer option because you are making sure all parameters are assigned one-by-one. I would not change this code because people's opinions might differ and we should keep the test code as similar as possible. Would you mind keeping the old way of an array assignment?

Reply:
@okurz - we should keep the test code `as similar as possible` to what? What other parameters? How it's safer? I got lost :)

@asmorodskyi - as I said - it's cosmetics, feel free to ignore my comment.

Reply:
I guess a better generic way would be to define a repo specific style, check for that with static code checks and harmonize the whole test code accordingly - outside this PR of course